recipes/libsakura/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "libsakura" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "f387d6b2f633eca163ffb824df24742c69a3473971c671d10f801b143d4127c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://svn.cv.nrao.edu/casa/devel/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - compile-fixes.patch
+    - osx.patch  # [osx]
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc  # [osx]
+    - cmake
+    - eigen
+    - fftw 3.3.*
+  run:
+    - fftw 3.3.*
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
+    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
+
+about:
+  home: http://alma.mtk.nao.ac.jp/j/
+  license: GPLv3
+  license_file: COPYING
+  summary: 'C++ library for fast image manipulations'
+
+extra:
+  recipe-maintainers:
+    pkgw
```

Comment:
The maintainers should be a list. So this should be prefixed with a `- `. Please submit a PR to the feedstock and ping core to merge it. Thanks.